### DISCUSSION TITLE
Sovereign Tech Fund

### DISCUSSION THREAD

**Discussion by epicfaace** [json-schema-org/community#360] [Governance and Policy]
Hi all 👋 wanted to share this opportunity -- https://sovereigntechfund.de/en/applications/ -- it looks like they're funding foundational internet protocols / standards / implementations such as OpenMLS (see https://sovereigntechfund.de/en/projects/openmls/). I think json schema could be a great candidate and it might be an opportunity to get some good work done on schemas / implementations / validators / etc. I don't know if we have the right governance structure set up to accept funds and do this kind of work though. What do you think?

**Comment by jviotti** [comment]
Hey @epicfaace , thanks for sharing! @benjagm @Relequestual This sounds like a very cool opportunity. I'd love to help applying. Maybe worth discussing on Monday? 

One potential idea of something I'd love to fund/lead is proper modern and compliant JSON Schema tooling for C++, as these kind of tooling can provide bindings to a big range of programming languages and support development of more advanced tooling like JSON BinPack (https://www.jsonbinpack.org)

**Comment by benjagm** [comment]
Thanks a lot for sharing @epicfaace. This looks amazing! 
I'll add this to today's Open Community Weekly Meeting to get feedback and decide the objectives of our application.
